Peek-swiftUI/MoviesTabBar.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  MoviesTabBar.swift
+//  Peek-swiftUI
+//
+//  Created by mazen eldeeb on 24/08/2024.
+//
+
+import SwiftUI
+
+struct MoviesTabBar: View {
+    @AppStorage("selectedTab") private var selectedTab = 0
+    
+    init() {
+        UITabBar.appearance().backgroundColor = .mainPurple
```

Comment:
make both the TabBar appearancne modifications in one functionn annd either call it in the init or in onAppear